operator/pkg/ciliumendpointslice/rate_limit.go

```diff
@@ -29,87 +30,68 @@ type rateLimitConfig struct {
 	logger logrus.FieldLogger
 }
 
+type dynamicRateLimit []rateLimit
+
+var defaultDynamicRateLimit = dynamicRateLimit{
+	{Nodes: 0, Limit: 10, Burst: 20},
+	{Nodes: 100, Limit: 7, Burst: 15},
+	{Nodes: 500, Limit: 5, Burst: 10},
+}
+
 func getRateLimitConfig(p params) rateLimitConfig {
-	var dynamicRateLimit []rateLimit
-	if p.Cfg.CESEnableDynamicRateLimit {
-		var err error
-		dynamicRateLimit, err = parseDynamicRateLimit(p.Cfg.CESDynamicRateLimitNodes, p.Cfg.CESDynamicRateLimitQPSLimit, p.Cfg.CESDynamicRateLimitQPSBurst)
+	dynamicRateLimit := defaultDynamicRateLimit
+	if len(p.Cfg.CESDynamicRateLimitConfig) != 0 {
+		parsed, err := parseDynamicRateLimit(p.Cfg.CESDynamicRateLimitConfig)
 		if err != nil {
 			p.Logger.WithError(err).Warn("Couldn't parse dynamic rate limit config")
```

Comment:
It's personally suggest to bail out loudly if the configuration cannot be parsed, to avoid surprises to users.

Reply:
Done :+1: